mappings/net/minecraft/block/IBlockInfoProxy.mapping

```diff
@@ -57,8 +57,13 @@ CLASS none/asl net/minecraft/block/IBlockInfoProxy
 		ARG 1 pos
 	METHOD e allowsLightTransmission ()Z
 	METHOD g getMapColor ()Lnone/ays;
-	METHOD h ()Z
+	METHOD h isFullBoundsCube2 ()Z
```

Comment:
Ended up just removing the mapping for now, it may make sense when more stuff is mapped.